extensions/hp-extension/src/commandsModule.tsx

```diff
@@ -0,0 +1,99 @@
+const commandsModule = ({ servicesManager, commandsManager }) => {
+  const {
+    DisplaySetService,
+    ViewportGridService,
+    HangingProtocolService,
+  } = servicesManager.services;
+
+  const actions = {
+    nextStage: () => {
+      // next stage in hanging protocols
+      HangingProtocolService.nextProtocolStage();
+    },
+    previousStage: () => {
+      HangingProtocolService.previousProtocolStage();
+    },
+
+    updateViewportDisplaySet: ({ direction, circular, sortSource }) => {
```

Comment:
is sortSource being used?

Reply:
Important of saying the expect value/values for each. For instance

direction = 1 or -1
circular = boolean

sortSource should be removed